Serializer/JsonApiSerializationVisitor.php

```diff
@@ -37,19 +34,22 @@ class JsonApiSerializationVisitor extends JsonSerializationVisitor
      */
     protected $showVersionInfo;
 
+    /**
+     * @var bool
+     */
     protected $isJsonApiDocument = false;
 
     /**
      * @param PropertyNamingStrategyInterface $propertyNamingStrategy
      * @param AccessorStrategyInterface       $accessorStrategy
      * @param MetadataFactoryInterface        $metadataFactory
-     * @param                                 $showVersionInfo
+     * @param bool                            $showVersionInfo
      */
     public function __construct(
         PropertyNamingStrategyInterface $propertyNamingStrategy,
         AccessorStrategyInterface $accessorStrategy = null,
         MetadataFactoryInterface $metadataFactory,
-        $showVersionInfo
+        $showVersionInfo = true
```

Comment:
I think it is better to be as `false`, actually. It has no big use in response :)

Reply:
@RuslanZavacky I pointed in in your pull request to change it to true, due to bundle configuration.
From configuration as default "true" is passed.